srcalc/src/main/java/gov/va/med/srcalc/security/VistaUserDetails.java

```diff
@@ -20,26 +20,23 @@
     
     private final VistaPerson fVistaPerson;
     
-    private final List<GrantedAuthority> fAuthorities;
-    
     /**
      * Constructs an instance.
      * @param vistaPerson the wrapped VistaPerson
-     * @param authorities the list of authorities for the user
      */
-    public VistaUserDetails(
-            final VistaPerson vistaPerson, final List<GrantedAuthority> authorities)
+    public VistaUserDetails(final VistaPerson vistaPerson)
     {
         fVistaPerson = vistaPerson;
-        fAuthorities = authorities;
     }
 
+    /**
+     * This is simple: all VistA users are treated alike, so this method always
+     * returns a one-element collection containing {@link Roles#ROLE_USER}.
+     */
```

Comment:
Will we need a class like VistaAdminDetails for a VistA admin, or is there no such thing?